### ISSUE TITLE
Github Desktop always running in the background

### ISSUE THREAD

**Issue by ninja542** [desktop/desktop#2002]
### Description

This isn't an issue, but I'm not sure if this is supposed to happen, but Github desktop is always running in the background and it is kind of annoying since the old Github desktop doesn't run in the background. 
(I checked with Task Manager)

### Version

**GitHub Desktop version:** 0.5.7 (I think, for some reason github isn't running after I tried to end some of its tasks)

**OS version:** Windows 10

uhm I'm not very github savvy, so I don't know where else I should put this.

**Comment by shiftkey** [comment]
Closing in favour of #1641 - please add details about your setup there

**Comment by shiftkey** [comment]
Actually, I should clarify the behaviour before pointing you there.

> Github desktop is always running in the background and it is kind of annoying since the old Github desktop doesn't run in the background.

@ninja542 is this related to processes hanging around after you close GitHub Desktop? Could you share some repro steps around what you are doing so we can understand the behaviour you are seeing?

**Comment by ninja542** [comment]
Yes, I have GitHub Desktop closed, and the processes still show up in Task Manager.

Before closing the window of GitHub Desktop, all I did was push some of my changes to one of my personal repositories on GitHub. I didn't do any git bashing? 

I also tried to link github to sublime text, but I don't think that is a factor, but I'm putting it here in case. 

Does this help?

**Comment by shiftkey** [comment]
@ninja542 that helps, I'll reopen this and see if this can be reproduced

**Comment by ninja542** [comment]
I also tried to end some of the tasks because I wanted to see if it would help, but then I couldn't open github desktop, so I have it uninstalled now and I think I'll just use the old client for now

**forgot to add this, sorry**

**Comment by tierninho** [comment]
Wondering if this is related? https://github.com/desktop/desktop/issues/1502#issuecomment-307450097

**Comment by ninja542** [comment]
the picture is related yes, but the issue itself is slightly different. I didn't toggle full screen or anything

**Comment by Mr-BrainSpace** [comment]
I had this same issue when I first installed, after closing the desktop application on Windows 7 I had processes running up the CPU like #1641 . At the same time a process called Update.exe was being suspicious. I found reopening Github Desktop, and going File > Close -- vs the usual red [X] -- got rid of both problems. Could be a coincidence.

![image](https://user-images.githubusercontent.com/30075973/28076139-b4b5dcfc-662b-11e7-894d-86ef454b93e6.png)
